CLogger/include/CLogger/CLogger.h

```diff
@@ -363,6 +363,10 @@ CLogger& CLogger::AddLogConfig(const ELogConfig log_config, Args... args)
 template<typename... Args>
 void CLogger::PrintLogMessage(const CLogMessage<Args...>& log_message) const
 {
+	if(this == nullptr)
```

Comment:
I do not think so that such kind of code should exist.

Reply:
Deleted